dev/constraints.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <cassert>
```

Comment:
Please remove

Reply:
Done, sorry for missing this one - I was just looking in `examples/` before.
As best I can tell, this PR no longer contains any new `#include <cassert>`:
```
$ git df origin/dev HEAD | grep cassert
<no output>
```